src/Database/QueryBuilder/MysqlQueryBuilder.php

```diff
@@ -217,7 +219,7 @@ private function createColumnDefault(Column $column): string
             if ($column->getType() === Column::TYPE_TIMESTAMP && $column->getSettings()->getDefault() === ColumnSettings::DEFAULT_VALUE_CURRENT_TIMESTAMP) {
                 return $default . 'CURRENT_TIMESTAMP';
             }
-            return $default . "'" . $column->getSettings()->getDefault() . "'";
+            return $default . "'" . str_replace("'", "\'", $column->getSettings()->getDefault()) . "'";
```

Comment:
I've look at it and found a problem. This is the fix for `'` in default value, but not for type BIT :) we have to store BIT's default value as it comes from settings - Same as for type INTEGER few lines above, so I suggest to do this:

change:
```
if ($column->getType() === Column::TYPE_INTEGER) {
    return $default . $column->getSettings()->getDefault();
}
```
to:
```
if (in_array($column->getType(), [Column::TYPE_INTEGER, Column::TYPE_BIT], true)) {
    return $default . $column->getSettings()->getDefault();
}
```
And it will set the correct value to database. Same fix we need also for pgsql adapter.

After this, we will have some troubles with NULLable bit values with default value. BIT needs to be handled similar to TIMESTAMP because its syntax is:
```
bit(32) NULL DEFAULT b'0'
```
So it will need some refactoring...

Will you try to do it or should I when I will have some time to it?

Reply:
I would appreciate it if you can. I have a baby at home and now there is not enough free time outside of work.

Reply:
Congratulation! I have also 1-month baby here, that’s why I have not too much time for open source. But I will try to find some and fix it